Improved help documentation (#330)
vgteam/sequenceTubeMap: public/help/help.md

#### Sequence Tube Map Usage Guide

> The Sequence Tube Map is used to generate visualizations of genomic sequence graphs. This guide will demonstrate how to select custom data to display visualizations. [Link to usage guide on Github](https://github.com/vgteam/sequenceTubeMap/blob/readme_updates/public/help/help.md)

Reviewer: ```suggestion
The Sequence Tube Map is used to generate visualizations of genomic sequence graphs. This guide will demonstrate how to select custom data to visualize. [You can also read this guide on Github.](https://github.com/vgteam/sequenceTubeMap/blob/master/public/help/help.md)
```

We probably want to link to where this is going to appear long-term, once we merge the PR, instead of to the branch.

##### Selecting Tracks

> The following procedure describes adding and updating settings of custom tracks.

Reviewer: ```suggestion
The following procedure describes adding and updating settings of custom tracks. You can use a custom track to load your own graph file, haplotype database, or file of aligned reads.
```

We should say a bit more about why one would do this.

> 1. From the "Data" dropdown bar, select "custom (mounted files)"  

Reviewer: ```suggestion
1. From the "Data" dropdown box, select "custom (mounted files)"  
```

I think this is really still a dropdown box and not actually a navbar.

![Selecting Custom Files](helpGuideImages/img1.png)  
2. Select the button that says "Configure Tracks".  
![Configure Tracks Button](helpGuideImages/img2.png)
3. Once the button is clicked on, a popup with a "+" button will be displayed. Click on this button to add tracks.  
![Track Add Button](helpGuideImages/img3.png)  

> 4. Select a graph, read, or haplotype track from the first dropdown. Make sure to always have at least 1 graph track. Select a data file adhering to the track type from the second dropdown.  

Reviewer: ```suggestion
4. Select a graph, read, or haplotype track from the first dropdown. Make sure to always have at least 1 graph track. Select a data file from the list of files of that type in the second dropdown.  
```

![Track Selection](helpGuideImages/img4.png)  
5. Click on the settings button, where there are options to color the tracks from an existing color palette, or select any other color.  
![Track Settings Button](helpGuideImages/img5.png)  
7. To delete a track, click on the button with the "x" icon.  
![Track Delete Button](helpGuideImages/img6.png)

##### Displaying Visualizations
1. Add as many tracks as needed and exit the track picker. 
2. Add a BED file, if needed. This step is optional.
![Bed File Selection](helpGuideImages/img7.png)

> 3. Specify a region input. The region input can be: a coordinate range (e.g. "chr1:1-100"), a node ID range (e.g. "node:100-110"), a start position and a distance (e.g. "chr1:1+100"), a node ID anchor and a distance (e.g. "node:100+10")

Reviewer: ```suggestion
3. Specify a region input. The region input can be:
   * A coordinate range (e.g. "chr1:1-100")
   * A node ID range (e.g. "node:100-110")
   * A start position and a distance (e.g. "chr1:1+100")
   * A node ID anchor and a distance (e.g. "node:100+10")
```

Since we have the space I think we can bullet these out. If we don't, we need the missing `or` in the sentence.

![Region Input Options](helpGuideImages/img8.png)

> 9. Click Go to see the selected tracks render in the graph.

Reviewer: ```suggestion
9. Click Go to see the selected tracks render in the visualization area.
```
